### ISSUE TITLE
Accessibility Fix for Featured Images as Links in Latest Posts Block

### ISSUE THREAD

**Issue by robin-scott** [WordPress/gutenberg#36250]
## What problem does this address?

As mentioned in https://github.com/WordPress/gutenberg/pull/24548#issuecomment-961753518 

There is an accessibility issue with the "add link to featured image" option within latest posts block -- in that "Links do not have a discernible name" & should be updated to include an aria-label="The post title" in there.

This option: 

![image](https://user-images.githubusercontent.com/17259256/140503859-2f84b847-3b39-4cdf-81eb-324916b8482e.png)

Produces accessibility warnings in Lighthouse (etc) "Links do not have a discernible name" - as our featured image links don't tell screen readers what the link is.

## What is your proposed solution?

I will add a branch / PR with the correct aria-label (I have already got this working!) - all that's needed is the title (which is used later on) variable to be declared a little earlier, then it can be included in this block.
